scss/responsive.scss

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-width: 1024px) {
+  .flex-body {
+
+    .train-header .dev{
+      display: none;
+    }
+
+    .train-main {
+      
+      .section .techs {
+        grid-auto-flow: row;
+        grid-template-columns: 1fr;
+
+        .tech {
+          background-color: transparent;
+          border-radius: 0;
+
+          &:first-child {
+            grid-column: auto;
+            grid-row: auto;
+            position: static;
+          }
+
+          &:not(:last-child) {
```

Comment:
This is quite complex, rather do `border-top` in tech and `border-top: 0` in `&:first-child`